src/workloads/transactions/LLTAnalytics.yml

```diff
@@ -0,0 +1,284 @@
+
+ActorTemplates:
+  - TemplateName: UpdateTemplate
+    Config:
+      Name: {^Parameter: {Name: "Name", Default: "Short.Update.Baseline"}}
+      Type: CrudActor
+      Threads: *ThreadsValue
+      Phases:
+        OnlyActiveInPhases:
+          Active: {^Parameter: {Name: "Active", Default: [5]}}
+          NopInPhasesUpTo: *MaxPhases
+          PhaseConfig:
+            GlobalRate: *GlobalRateValue
+            Threads: *ThreadsValue
+            CollectionCount: *CollectionCount
+            Database: *dbname
+            Duration: {^Parameter: {Name: "Duration", Default: *SnapshotScannerShortDuration}}
+            Blocking:  {^Parameter: {Name: "Blocking", Default: yes}}
+            Operations:
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+              - *PtcUpdateOperation
+              - *PcUpdateOperation
+              - *CpcUpdateOperation
+
+  - TemplateName: ScanTemplate
+    Config:
+      Name: {^Parameter: {Name: "Name", Default: "Short.Scan.Snapshot"}}
+      Type: CollectionScanner
+      Threads: *CollectionCount
+      CollectionCount: *CollectionCount
+      Database: *dbname
+      GenerateCollectionNames: true
+      Phases:
+        OnlyActiveInPhases:
+          Active: {^Parameter: {Name: "Active", Default: [7]}}# [7]
+          NopInPhasesUpTo: *MaxPhases
+          PhaseConfig:
+            Duration: {^Parameter: {Name: "Duration", Default: *SnapshotScannerShortDuration}}
+            #ScanDuration: {^Parameter: {Name: "ScanDuration", Default: *SnapshotScannerShortDuration}}
+            ScanType: snapshot
+            #ScanContinuous: true
+            # GenerateCollectionNames: true
+            AggregatePipeline:
+              {array: [{"$match":{"cuid":{"$gte":500}}},
+                {"$unwind": "$orders"},
+                {"$group": {"_id": "$ts", "total": {"$sum":{ $multiply: [ "$orders.unitPrice", "$orders.quantity" ] }}}},
+                {"$sort": {"_id":1}}]
+              }
+            AggregateOptions:
+              BatchSize: 1000
+              Comment: {^Parameter: {Name: "Comment", Default: "Aggregate.Snapshot"}}
+              AllowDiskUse: true # Naming convention follows c++ driver rather than mongo shell.  
```

Comment:
is this needed, I suspect it is based on the group by ts and the size of the collections.